x-pack/plugins/actions/server/action_type_registry.test.ts

```diff
@@ -627,6 +647,35 @@ describe('actionTypeRegistry', () => {
         notifyUsage: true,
       });
     });
+
+    test('should return true for enabled type', async () => {
+      mockedLicenseState.isLicenseValidForActionType.mockReturnValue({ isValid: true });
+      const result = actionTypeRegistry.isActionExecutable('123', 'foo');
+      expect(result).toEqual(true);
+    });
+
+    test('should return false when license invalid', async () => {
+      mockedLicenseState.isLicenseValidForActionType.mockReturnValue({
+        isValid: false,
+        reason: 'invalid',
+      });
+      const result = actionTypeRegistry.isActionExecutable('123', 'foo');
+      expect(result).toEqual(false);
+    });
+
+    test('should return true for disabled type, but preconfigured connector', async () => {
```

Comment:
adding some tests for our forgotten but suprising "you can run preconfigured connectors even if the connector type is disabled - both are done via config"